node_modules/col-assets/lib/api.js

```diff
@@ -267,6 +273,90 @@ var createLink = module.exports.createLink = function(ctx, title, url, opts, cal
   });
 };
 
+/**
+ * Edit an asset
+ *
+ * @param  {Context}        ctx                             Standard context containing the current user and the current course
+ * @param  {Number}         id                              The id of the asset that is being edited
+ * @param  {String}         title                           The updated title of the asset
+ * @param  {Object}         [opts]                          A set of optional parameters
+ * @param  {Number[]}       [opts.categories]               The updated ids of the categories to which the asset should be associated
```

Comment:
This is a bit misleading. If no `opts.categories` are provided then the asset will end up with 0 categories as we're doing `asset.setCategories`